src/pages/_app.js

```diff
@@ -38,16 +41,18 @@ class MyApp extends App {
 
     return (
       <>
-        <UserContext.Provider value={{ user: this.props.userDetails }}>
-          <Layout serviceName="Repairs Hub">
-            <Meta />
-
-            <ComponentToRender
-              {...pageProps}
-              userDetails={this.props.userDetails}
-            />
-          </Layout>
-        </UserContext.Provider>
+        <QueryClientProvider client={queryClient}>
+          <UserContext.Provider value={{ user: this.props.userDetails }}>
+            <Layout serviceName="Repairs Hub">
+              <Meta />
+
+              <ComponentToRender
+                {...pageProps}
+                userDetails={this.props.userDetails}
+              />
+            </Layout>
+          </UserContext.Provider>
+        </QueryClientProvider>
```

Comment:
Like I said above, this should be limited to backoffice.

Reply:
All of the frontend requests use Axios. It would add more complexity by varying how we fetch this data. Im not opposed to trying something different, but we should think about the implications before we roll it out everywhere.